experimental/packages/propagator-aws-xray-lambda/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export * from './AWSXRayLambdaPropagator';
```

Comment:
Let's avoid `export *` so that future changes to the public API are more visible (also avoids exporting `AWSXRAY_TRACE_ID_ENV_VAR`)

```suggestion
export { AWSXRayLambdaPropagator } from './AWSXRayLambdaPropagator';
```